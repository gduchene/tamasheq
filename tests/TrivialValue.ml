let x = 1
let y = x + 1
