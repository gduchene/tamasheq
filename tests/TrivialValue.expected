TrivialValue_0	=	[2] {
    (no name)	=	1
    (no name)	=	2
}
